deploy/backplane/cee/30-srep.SubjectPermission.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-cee-readers
+  namespace: openshift-rbac-permissions
+spec:
+  clusterPermissions:
+  - backplane-cee-readers-cluster
+  permissions:
+  - allowFirst: true
+    clusterRoleName: "openshift-cee-mustgather"
```

Comment:
`openshift-mustgather`? or should the clusterrole be called `openshift-cee-mustgather`?

Reply:
Missed a commit, repushed